katas/es6/language/generator/iteration.js

```diff
@@ -0,0 +1,43 @@
+// 50: Generators return iterable objects
+// To do: make all tests pass, leave the assert lines unchanged!
+
+describe('a generator returns an iterable object', function() {
+  
+  const gen = function* (){
+    yield 1;
+    yield 2;
+  };
+  
+  let genInstance;
+  
+  beforeEach(() => {
+    genInstance = new gen();
```

Comment:
I made the kata, and I think it does reflect well what this is about! Just tiny things and I would merge it for tomorrow.
`genInstance = new gen();` better names could be `generator = new generatorFunction();`